DESCRIPTION

```diff
@@ -21,6 +21,7 @@ LazyData: true
 Imports:
     stringr,
     lubridate,
+    tibble,
```

Comment:
We should add some version requirements on these packages, particularly for the tidyverse ones, which have had some behavior changes in the past.  I'll knock those out in just a moment her.